src/DIRACbenchmark/dirac_benchmark.py

```diff
@@ -0,0 +1,277 @@
+
+""" DIRAC Benchmark 2012 by Ricardo Graciani, and wrapper functions to
+    run multiple copies in parallel by Andrew McNab.
+    This file (dirac_benchmark.py) is intended to be the ultimate upstream
+    shared by different users of the DIRAC Benchmark 2012 (DB12). The
+    canonical version can be found at https://github.com/DIRACGrid/DB12
+    This script can either be imported or run from the command line:
+    ./dirac_benchmark.py NUMBER
+    where NUMBER gives the number of benchmark processes to run in parallel.
+    Run  ./dirac_benchmark.py help  to see more options.
+"""
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+
+import os
+import sys
+import random
+import multiprocessing
+
+VERSION = "1.0.0 DB12"
```

Comment:
@fstagni should we keep the version number here? 

We rely on `setuptools-scm` now.
We can probably get the version from `pkg_resources` like in this example:
https://github.com/pypa/setuptools_scm#retrieving-package-version-at-runtime

However, there is a disadvantage:
```
However, this does place a runtime dependency on setuptools 
and can add up to a few 100ms overhead for the package import time.
```